Comanche055/P76.agc

```diff
@@ -35,10 +35,10 @@
 #	TIME (TIG) AT WHICH THE OTHER VEHICLE VELOCITY WAS CHANGED BY EXECUTION OF A THRUSTING MANEUVER. THE
 #	OTHER VEHICLE STATE VECTOR IS INTEGRATED TO TIG AND UPDATED BY THE ADDITION OF DELTA V (DELTA V HAVING
 #	BEEN TRANSFORMED FROM LV TO REF COSYS).  USING INTEGRVS, THE PROGRAM THEN INTEGRATES THE OTHER
-#	VEHICLE STATE VECTOR TO THE STATE VECTOR OF THIS VEHICLE, THUS INSURING THAT THE W-MATRIX AND BOTH VEHICLE
+#	VEHICLE STATE VECTOR TO THE STATE VECTOR OF THIS VEHICLE. THUS INSURING THAT THE W-MATRIX AND BOTH VEHICLE
 #	STATES CORRESPOND TO THE SAME TIME.
 # 3)	ERASABLE INITIALIZATION REQUIRED -- NONE.
-# 4)	CALLING SEQUENCES AND EXIT MODES -- CALLED BY ASTRONAUT REQUEST THRU DSKY V 37 E 76E.
+# 4)	CALLING SEQUENCES AND EXIT MODES -- CALLED BY ASTRONAUT REQUEST THRU DSKY V 37 E 76 E.
 #	EXITS BY TCF ENDOFJOB.
 # 5)	OUTPUT -- OTHER VEHICLE STATE VECTOR INTEGRATED TO TIG AND INCREMENTED BY DELTA V IN REF COSYS.
 #	THE PUSHLIST CONTAINS THE MATRIX BY WHICH THE INPUT DELTA V MUST BE POST-MULTIPLIED TO CONVERT FROM LV
```

Comment:
For line `# 7)...`

✨ No spacing between list of `BANKCALL` to `CSMPREC`
✨ No spacing at `LEMPREC),ATOPCSM`
✨ No spacing between list of `ATOPLEM)` to `INTWAKE`

For line `# 8)...`

✨ No spacing between list of `MOONFLAG` to `RASFLAG`